src/test/java/ay2021s1_cs2103_w16_3/finesse/testutil/TransactionUtil.java

```diff
@@ -26,6 +28,13 @@ public static String getAddCommand(Transaction transaction) {
         return AddCommand.COMMAND_WORD + " " + getTransactionDetails(transaction);
     }
 
+    /**
+     * Returns an add expense command string for adding the {@code transaction}.
+     */
+    public static String getAddExpenseCommand(Expense expense) {
+        return AddExpenseCommand.COMMAND_WORD + " " + getTransactionDetails(expense);
+    }
+
     /**
      * Returns an add income command string for adding the {@code transaction}.
```

Comment:
This should be changed as well, my bad!
```suggestion
     * Returns an add income command string for adding the {@code income}.
```